plot_multi_waveforms.py

```diff
@@ -0,0 +1,28 @@
+import make_waveforms
+import pylab as pl
+import numpy as np
+
+
+hp,hc= make_waveforms.make_tidal_waveform(mass1= 10, mass2 = 8, s1z=0, approx = "SpinTaylorT4")
+
+hp2,hc2=make_waveforms.make_tidal_waveform(mass1= 10, mass2 = 8, s1z = 0.5, approx = "SpinTaylorT4")
+
+hp3,hc3=make_waveforms.make_tidal_waveform(mass1 = 10, mass2 = 8, s1z = -0.5, approx = "SpinTaylorT4")
+
+def get_times(hp):
+    t=[]
+    for t in range (0,10):
+        t=np.arange(0, -hp.epoch.gpsSeconds - hp.epoch.gpsNanoSeconds*1e-9 + hp.deltaT, hp.deltaT)
+    t=t[:hp.data.length]
+    return t
```

Comment:
You have initialized a list `t`, but then you wrote `for t in range (0,10):`, which will overwrite the empty list `t` by the iterating integer. Then inside the loop you are creating a `numpy` array `t` which is the actual array of time. Finally, you are cropping the array up to the length of the waveform. So the first two `t` are actually never used. Your for loop is doing nothing, except the same computation `t=np.arange(0, -hp.epoch.gpsSeconds - hp.epoch.gpsNanoSeconds*1e-9 + hp.deltaT, hp.deltaT)` is repeated 10 times. This whole thing can be replaced by this:
```
def get_times(hp):
    t = np.arange(0, -hp.epoch.gpsSeconds - hp.epoch.gpsNanoSeconds*1e-9 + hp.deltaT, hp.deltaT)
    return t[:hp.data.length]
```